### ISSUE TITLE
How to perform Batch wise training for larger dataset

### ISSUE THREAD

**Issue by jagabandhumishra** [keras-team/keras#11353]
I have training data chunked into 5 parts, which are large in size, I cant load them at a time, I like to batch wise train the model by loading and unloading every chunk of data, Is there any method in keras to do so, Please suggest.

**Comment by bono1567** [comment]
For a Sequential keras model you can train batchwise using this command after the compile command
`model.fit(data,label,epochs=n_of_epochs,batch_size=size)`
